src/HttpHeader.cc

```diff
@@ -486,6 +488,15 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
             return 0;
         }
 
+        const bool frameHeader = (e->id == Http::HdrType::CONTENT_LENGTH || e->id == Http::HdrType::TRANSFER_ENCODING);
```

Comment:
Nit: I think "framing header" sounds more standard/accurate. I do not insist on this change.

```suggestion
        const auto framingHeader = (e->id == Http::HdrType::CONTENT_LENGTH || e->id == Http::HdrType::TRANSFER_ENCODING);
```

or even 

```suggestion
        const bool framing = (e->id == Http::HdrType::CONTENT_LENGTH || e->id == Http::HdrType::TRANSFER_ENCODING);
```

Reply:
Done in 1a826d532